conanfile.py

```diff
@@ -36,6 +36,7 @@ def set_version(self):
 
     def requirements(self):
         self.requires("boost/[<1.70]", override=True)
+        self.requires("fmt/[~=6.2.0]", override=True)
```

Comment:
Why not use the spdlog (v1.5.0) fmt-dependency restriction here?
`self.requires("fmt/[<7.0]", override=True)`

Reply:
I removed the fmt version override from the cloe meta package since there is currently no conflict between the spdlog and the fable fmt dependencies. If this changes in the future, we can add the override.